src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt

```diff
@@ -70,6 +75,20 @@ class AcquirersService(
     ).toList()
   }
 
+  fun deleteAcquirerSubscriptionEnrichmentField(enrichmentFieldId: UUID) {
+    val now = dateTimeHandler.now()
+    adminActionAlertsService.noticeAction(
+      AdminAction(
+        "Delete acquirer subscription enrichment field",
+        object {
+          val enrichmentFieldId = enrichmentFieldId
+          val whenDeleted = now
+        },
+      ),
+    )
+    acquirerSubscriptionEnrichmentFieldRepository.deleteById(enrichmentFieldId)
```

Comment:
Why are we hard deleting these and soft deleting everything else?

Reply:
discussed